src/main/java/frc/robot/RobotContainer.java

```diff
@@ -78,6 +82,10 @@ public RobotContainer() {
     configureBindings();
     initAutoChooser();
 
+    NamedCommands.registerCommand("AutoRunIntake", new AutoRunIntake(m_intake, -0.5, -0.85));
```

Comment:
Should make the intake speeds a variable in constants or something, so they're easier to debug while testing autos